src/containers/HeaderContainer/HeaderContainer.vue

```diff
@@ -89,6 +89,24 @@
                 <b-nav-item v-if="isHomePage" to="/#about-mew">{{
                   $t('header.about')
                 }}</b-nav-item>
+                <b-nav-item-dropdown
+                  v-if="wallet !== null"
+                  right
+                  no-caret
+                  class="tx-history-menu"
+                >
+                  <template slot="button-content">
+                    <p>Transaction History</p>
+                  </template>
+                  <b-dropdown-item :href="blockExplorerURL" target="_blank">
+                    <p>Etherscan (ETH)</p></b-dropdown-item
+                  >
+                  <b-dropdown-item
+                    :href="'https://ethplorer.io/address/' + address"
+                    target="_blank"
+                    >Ethplorer (Tokens)</b-dropdown-item
```

Comment:
Ethplorer is only available on `ETH` hide if not ETH

Reply:
Pull `devop/prep-release` so you can use `network.currencyName`